DependencyInjection/SuluAttributesExtension.php

```diff
@@ -16,20 +19,19 @@
 class SuluAttributesExtension extends Extension implements PrependExtensionInterface
 {
     use PersistenceExtensionTrait;
+
     /**
      * Allow an extension to prepend the extension configurations.
      */
     public function prepend(ContainerBuilder $container): void
     {
     }
-    /**
-     * {@inheritdoc}
-     */
+
     public function load(array $configs, ContainerBuilder $container): void
     {
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
-        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
```

Comment:
Here is a bug that I caused. The services.yml is not in the correct folder as correctly specified here

Reply:
Fixed.